document-message/document-message.bal

```diff
@@ -0,0 +1,30 @@
+import ballerina/http;
+import ballerina/mime;
+
+type CsvRequest record {
+    string path;
+    string CRM_ORG;
+    string feature;
+};
+
+type ZohoResponse record {
+    string status;
+    string code;
+    string message;
+    record {|
+        string file_id;
+        string created_time;
+    |} details;
+};
+
+service /api on new http:Listener(8080) {
+    resource function post uploadCsv(CsvRequest csvRequest) returns ZohoResponse|error {
```

Comment:
```suggestion
    resource function post uploadFtpCsv(CsvRequest csvRequest) returns ZohoResponse|error {
```
What about `uploadFtpCsv`

Reply:
I think uploadCsv is better. Since otherwise it looks like we are uploading *to* ftp.